main.py

```diff
@@ -298,6 +313,29 @@ def analyse_release(release: dict, project_name: str) -> Optional[dict]:
     }
 
 
+def analyse_tag(release: dict, project_info: dict) -> Optional[dict]:
+    """ Heuristics to find the version number """
+    project_name = project_info["name"]
+    match_name = extract_version(release.get("name") or "", project_name)
+    if match_name is not None:
+        release_type, version = match_name
+    else:
+        logger.warning("Invalid version strings '{}'".format(release["name"]))
+        return None
+
+    date = string_to_wddate(
+        get_json_cached(release["commit"]["url"])["commit"]["committer"]["date"]
+    )
+    html_url = project_info["html_url"] + "/releases/tag/" + release["name"]
```

Comment:
Is `release["name"]` guaranteed to be urlsafe? Otherwise it should be `urlencode`d.